lib/jest-helpers.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+const nunjucks = require('nunjucks')
+const cheerio = require('cheerio')
+const yaml = require('js-yaml')
+
+const COMPONENT_DIR = 'src/components'
```

Comment:
Missed this one, would've been good to use `config/paths.json` so we have all the paths in one place.

Reply:
Will do a PR